### PR TITLE
feat: Editable directions in state machine

### PR DESCRIPTION
**Pull request by hannahpurcell**
Asana Ticket: https://app.asana.com/0/1205385723132845/1208153509160218

Builds off of 
- https://github.com/mbta/skate/pull/2879

### CODE REVIEW

**Comment by firestack** [comment]
I'm hesitant to merge this before we're about to make changes to how detour directions are edited, so we can avoid any backend migrations, does this seem like an issue to you?

**Comment by hannahpurcell** [comment]
> I'm hesitant to merge this before we're about to make changes to how detour directions are edited, so we can avoid any backend migrations, does this seem like an issue to you?

Ahhhh definitely an issue. Would this resolve our problem: I add a frontend / state-machine handler for when `editedDirections` was not previously initialized, that field should be initialized when the active / past detour is opened?

**Review by firestack — APPROVED** [review]

File: assets/css/_diversion_page.scss [hunk 1]
```diff
@@ -116,6 +116,10 @@
     flex: 1 1 0;
     overflow-y: auto;
   }
+
+  .form-control {
+    min-height: 200px;
+  }
```
[the remark this answers is not in the document]
Comment (hannahpurcell) [reply assets/css/_diversion_page.scss:122]: https://github.com/mbta/skate/pull/2883

**Comment by hannahpurcell** [comment]
> Ahhhh definitely an issue.

[Fixed here](https://github.com/mbta/skate/pull/2880/commits/246acdf7ee00cc320d1d9feca2850d2c023d0078)

**Review by firestack — APPROVED** [review]

**Pull request merged**